### PR TITLE
Switched columns in Price List page to show more user-friendly information.

### PR DESCRIPTION
**Pull request by jhuseman**
Rental is now "yes" or "no"
Price now always 2 decimals, right aligned

### CODE REVIEW

**Comment by jhuseman** [comment]
Simple cosmetic change - should be ready for review.

**Comment by jhuseman** [comment]
I checked all the lines I edited to make sure they use spaces instead of tabs.  I left everything else alone, so hopefully that will avoid causing merge conflicts with my other pull request, and have spaces for the lines changed here.

File: bookfinder/static/js/prices.js [hunk 1]
```diff
@@ -87,7 +87,13 @@ function add_option_to_list(option, list_id) {
 	//Rental:
 	var new_rental = document.createElement("td");
 	var new_rental_link = document.createElement("a");
-	var new_rental_text = document.createTextNode(option.rental);
+    var rental_str = ""; // variable created here for scope
+    if(option.rental=="true") {
+        rental_str = "yes";
```
Comment (cshinaver) [inline bookfinder/static/js/prices.js]: Uppercase for niceness

File: bookfinder/templates/search/price-comparison.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 					<th>Seller</th>
 					<th>Type</th>
 					<th>Rental</th>
-					<th>Price</th>
+                    <th><span style="float:right;">Price</span></th>
```
Comment (cshinaver) [inline bookfinder/templates/search/price-comparison.html]: `http://www.w3schools.com/tags/att_div_align.asp`

**Comment by cshinaver** [comment]
approved

**Pull request merged**